src/app/providers/data.service.ts

```diff
@@ -52,6 +52,15 @@ export class DataService {
                          .catch(this.handleNotFound);
     }
 
+    public openBoard(){
```

Comment:
I think it would make more sense for this to be put in the ElectronService, in which case you would need to inject it into the NavbarComponent and pass in the current active board via getActiveBoard from the DataService. The DataService is currently really bloated with a lot of different functionality, which is something that I will refactor soon.

Reply:
yeah you're right, moved it to electron service. Not sure if i did the injecting thing right, just looked at what it used for dataservice and copied that.